tests/hardware_tests/configs/waypoints.yaml

```diff
@@ -5,6 +5,19 @@ place_targets:
   - 0.0000000000000000
   - 0.7500000000000000
 
+  test_place_f:
```

Comment:
Are these test places in the global coordinate?

Reply:
Yes Jimmy